qemu/tests/blockdev_stream_cor_base.py

```diff
@@ -43,6 +44,11 @@ def _is_backing_exit(backing, depth):
         image = self.get_image_by_tag(name)
         filename = image.image_filename
         is_cor = backing["driver"] == "copy-on-read"
+        backing_mask = self.main_vm.check_capability(
+                Flags.BLOCKJOB_BACKING_MASK_PROTOCOL)
+        raw_format = image.image_format == "raw"
+        raw_elimi = backing_mask and raw_format
+        is_cor = is_cor and not raw_elimi
```

Comment:
`
opts = backing["file"]["file"] if (is_cor and not raw_elimi) else backing["file"]
`
What about moving the check here, because is_cor is True once copy-on-read is defined, no matter if the raw format node is eliminated

Reply:
for raw format elimination, we just needs opts = backing["file"]; and for other condition, opts = backing["file"]["file"]